### PR TITLE
Fix CI Errors

### PR DESCRIPTION
**Pull request by kchason**
- Suffix the artifacts with the Python version to avoid conflicts

### CODE REVIEW

File: .github/workflows/prerelease.yml [hunk 1]
```diff
@@ -33,10 +33,15 @@ jobs:
           - '3.12'
 
     steps:
-    - uses: actions/checkout@v3
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
```
Comment (ajnelson-nist) [inline .github/workflows/prerelease.yml:39]: Is the shallow clone necessary here, or was this a copy-paste artifact?

Comment (kchason) [reply .github/workflows/prerelease.yml:39]: 0 should be unlimited, which was necessary to get the submodules from some of the errors I was resolving

**Comment by ajnelson-nist** [comment]
Oh, also, this was started from `main` and targets `main`, when it should have started from and targeted `develop`.

File: .github/workflows/prerelease.yml (hunk 1 above)
Comment (ajnelson-nist) [reply .github/workflows/prerelease.yml:39]: You're right, my mistake.

**Pull request closed without merging**